superdesk/data_updates/00022_20200309-121947_archive.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : GyanP
+# Creation: 2020-03-09 12:19
+
+from bson import ObjectId
+from bson.errors import InvalidId
+from superdesk import get_resource_service
+from superdesk.vocabularies import is_related_content
+from superdesk.commands.data_updates import DataUpdate
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'archive'
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        related = list(get_resource_service('vocabularies').get(req=None, lookup={'field_type': 'related_content'}))
```

Comment:
this is redundant, it's done in `is_related_content` if `related_content` arg is not provided.

Reply:
good catch.I didn't looked at `is_related_content` definition.